docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3292,8 +3292,9 @@ table t {
 
 When programming table `t` in the example above, a P4Runtime client should
 specify the `TableAction` in the `TableEntry` to be a reference to either an
-action profile member or group. The reference is a `uint32` identifier that
-uniquely identifies a member or group programmed in the action selector `as`.
+action profile member or group. The reference is a non-0 `uint32` identifier
```

Comment:
I think this is more commonly spelled non-zero. I don't feel super strongly though.